openpype/modules/ftrack/plugins/publish/integrate_ftrack_status.py

```diff
@@ -0,0 +1,435 @@
+import copy
+
+import pyblish.api
+from openpype.lib import filter_profiles
+
+
+def create_chunks(iterable, chunk_size=None):
+    """Separate iterable into multiple chunks by size.
+
+    Args:
+        iterable(list|tuple|set): Object that will be separated into chunks.
+        chunk_size(int): Size of one chunk. Default value is 200.
+
+    Returns:
+        list<list>: Chunked items.
+    """
+    chunks = []
+
+    tupled_iterable = tuple(iterable)
+    if not tupled_iterable:
+        return chunks
+    iterable_size = len(tupled_iterable)
+    if chunk_size is None:
+        chunk_size = 200
+
+    if chunk_size < 1:
+        chunk_size = 1
+
+    for idx in range(0, iterable_size, chunk_size):
+        chunks.append(tupled_iterable[idx:idx + chunk_size])
+    return chunks
```

Comment:
This should be in a lib I suspect. There's an entry [here](https://github.com/ynput/OpenPype/blob/5b163fe84d49719c0ebfc0e47142688dd551e647/openpype/modules/ftrack/lib/avalon_sync.py#L137-L161). It being in `avalon_sync` might feel weird but maybe that just means that that function needs to go into `lib` to begin with?